stories/constants.js

```diff
@@ -8,7 +19,26 @@ export const icons = {
   Down,
   Left,
   Right,
-  Download
+  Download,
+};
+
+const makePrefix = (Icon) => <div className="pl-2">{<Icon />}</div>;
```

Comment:
I think the padding should be applied on the `EmailInput` component itself like we have in the `Input` component.